trailblazer/dto/create_analysis_request.py

```diff
@@ -0,0 +1,14 @@
+from pydantic import BaseModel
+
+from trailblazer.constants import JobType, TrailblazerPriority, TrailblazerTypes, WorkflowManager
+
+
+class CreateAnalysisRequest(BaseModel):
+    case_id: str
+    email: str | None = None
+    config_path: str
+    out_dir: str
+    priority: TrailblazerPriority
+    data_analysis: TrailblazerTypes
+    ticket_id: str | None = None
+    workflow_manager: WorkflowManager | None = None
```

Comment:
Did I miss any restrictions here? Maybe some of the fields should not be allowed to be None?